tests/versioned/transaction-naming-tests.js

```diff
@@ -149,6 +149,38 @@ function createTransactionTests(t, frameworkName) {
     })
   })
 
+  t.test('named query, multi-level with aliases should ignore aliases in naming', (t) => {
```

Comment:
should we also include this in the apollo-server-lambda transaction-naming suite?

Reply:
it's also prob worth adding this query to the segments test and asserting the names as they do take on the alias names for the segment path

Reply:
Good call, I'll add those tests.